screwdriver.yaml

```diff
@@ -1,19 +1,15 @@
-workflow:
-    - publish
-
 shared:
-    image: node:6
+    image: node:8
 
 jobs:
     main:
+        requires: [~pr, ~commit]
         steps:
             - install: npm install
             - test: npm test
-
     publish:
-        steps:
-            - install: npm install --no-save semantic-release@^7
-            - publish: npm run semantic-release
+        requires: main
+        template: screwdriver-cd/semantic-release-node8@1.0.0
```

Comment:
this template doesn't work and we haven't got time to look into this, you can still use the old template https://github.com/screwdriver-cd/models/blob/master/screwdriver.yaml#L13 to publish

Reply:
Done! 😄